Source/gg2/Sprites/HUDs/Gamemode HUDS/Arena/ArenaPlayerCountS.xml

```diff
@@ -1,9 +1,9 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
 <sprite>
-  <origin x="16" y="7"/>
+  <origin x="18" y="7"/>
   <mask>
     <separate>false</separate>
-    <shape>RECTANGLE</shape>
+    <shape>PRECISE</shape>
```

Comment:
I did not even realize the mask changed until now. Do more precise masks impact performance on things which do not collide? Should I switch this back to a rectangle mask?

Reply:
I wouldn't think it makes a difference here - not 100% sure, but I would guess that all collisions are done using bitmaps anyway in the end